pkg/ipam/ipam.go

```diff
@@ -394,7 +394,7 @@ func (liqoIPAM *IPAM) UnmapNetworkCIDR(_ context.Context, unmapCIDRRequest *Unma
 }
 
 // GetOrSetExternalCIDR get or set the external CIDR (eventually remapped) for the cluster.
-func (liqoIPAM *IPAM) GetOrSetExternalCIDR(_ context.Context, getOrSetExtCIDRRequest *GetOrSetExtCIDRRequest) (*GetOrSetExtCIDRResponse, error) {
+func (liqoIPAM *IPAM) GetOrSetExternalCIDR(ctx context.Context, getOrSetExtCIDRRequest *GetOrSetExtCIDRRequest) (*GetOrSetExtCIDRResponse, error) {
```

Comment:
Did you check that this function is idempotent and that multiple calls form the controller do not cause infinite error?

Reply:
Yes it is, I restarted multiple times the controller-manager, and I haven't encountered any error